src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
+import { User } from '../../types/User';
+import { Comment } from '../../types/Comment';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  title: string,
+  user: User,
+  body: string,
+  comments: Comment[] | null,
+};
+
+export const PostInfo: React.FC<Props> = ({
+  title,
+  user,
+  body,
+  comments,
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
```

Comment:
It's recommended to use a template string instead of concatenating strings with variables.